what4/src/What4/Interface.hs

```diff
@@ -1117,7 +1121,7 @@ class ( IsExpr (SymExpr sym), HashableF (SymExpr sym), HashableF (BoundVar sym)
   -- | Return the bitvector of the desired width with all 0 bits;
   --   this is the minimum unsigned integer.
   minUnsignedBV :: (1 <= w) => sym -> NatRepr w -> IO (SymBV sym w)
-  minUnsignedBV sym w = bvLit sym w (BV.zero w)
+  minUnsignedBV sym w = bvZero sym w
```

Comment:
Eek, I don't think you meant to change this part! `minUnsigned` and `bvZero` are now defined mutually in terms of each other, so calling either function will result in an infinite loop. (The CI can testify to this.)

I suppose this means that the `.hlint.yaml` file will need to carve out an exception for this module.

Reply:
Whoops, yep! This was a regex search/replace :sweat:

Reply:
For some reason, `hlint` doesn't seem to tag this nor `bvOne`. That seems fine, false negatives aren't a big problem for this kind of lint.